### PR TITLE
6-api-documentation

### PR DESCRIPTION
**Pull request by z6wdc**
I have followed these two documents to reorg our api and generate automatically documentation.
And we can see our documentation by our end point URL.
https://flask-restplus.readthedocs.io/en/stable/scaling.html
https://flask-restplus.readthedocs.io/en/stable/swagger.html

This PR also included  a `/member` api that will return the member's json file.
But I have not connected with our SQL.

I will create another PR for connecting our SQL and write a model for dealing with json file.

### CODE REVIEW

File: apis/cat.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    'id': fields.String(required=True, description='The cat identifier'),
+    'name': fields.String(required=True, description='The cat name'),
+})
+
+CATS = [
+    {'id': 'felix', 'name': 'Felix'},
+]
+
+@api.route('/')
+class CatList(Resource):
```
Comment (AsukaCHikaru) [inline apis/cat.py]: What's `cat` standing for?

File: apis/cat.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    @api.doc('list_cats')
+    @api.marshal_list_with(cat)
+    def get(self):
+        '''List all cats'''
+        return CATS
```
Comment (Ananpipi) [inline apis/cat.py]: This one?!

File: apis/cat.py (hunk 1 above)
Comment (z6wdc) [reply apis/cat.py]: It's a sample code from 
https://flask-restplus.readthedocs.io/en/stable/scaling.html.
And I just copied it without any revision.

File: main.py [hunk 3]
```diff
@@ -65,15 +65,6 @@ def notify():
         logger.info(message)
         requests.post(slack_webhook_url, json={'text': message})
 
-
-# TODO:
-# We should create a package and move this into that package
-@app.route('/photo', methods=['GET'])
```
Comment (dwi2) [inline main.py:71]: This means after this patch, `/photo` is no longer working.
Can we also add an `apis/photo.py` to maintain compatibility?

Comment (z6wdc) [reply main.py:71]: OK, I will add it.

Comment (z6wdc) [reply main.py:71]: added by this commit 971363cdb4a19f5663f18de5731291081d5b38cf

**Review by AsukaCHikaru — APPROVED** [review]
lgtm

**Review by dwi2 — APPROVED** [review]

**Pull request merged**